src/views/tickets/my-tickets/orders-list.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * My Tickets: Orders List
+ * 
+ * Override this template in your own theme by creating a file at [your-theme]/tribe-events/tickets/my-tickets/orders-list.php
+ * 
+ * @since TBD
+ * 
+ * @version TBD
+ * 
+ * @var  array  $orders  The orders for the current user.
+ * @var  int    $post_id The ID of the post the tickets are for.
+ */
+
+
+
+
+
+?>
+<ul class="tribe-orders-list">
+		<input type="hidden" name="event_id" value="<?php echo absint( $post_id ); ?>">
+		<?php foreach ( $orders as $order_id => $attendees ) : ?>
+			<?php
+
+			// Get provider from first attendee.
+			$first_attendee = reset( $attendees );
+
+			/** @var $provider Tribe__Tickets__Tickets */
+			$provider = Tribe__Tickets__Tickets::get_ticket_provider_instance( $first_attendee['provider'] );
+			if ( empty( $provider ) || ! method_exists( $provider, 'get_order_data' ) ) {
+				continue;
+			}
+			$order = $provider->get_order_data( $order_id );
+
+			?>
```

Comment:
If possible, can we move these logics into the main loader file? and pass them as variables instead? It will keep the template much lighter.

Reply:
Are you specifically talking about the `$provider` or are you also talking about passing in the order data (`$order`)?

Reply:
Ideally, we like to keep the logic away from templates by structuring them before the template is included, but as this has been like this previously and we are not transferring to `v2` templates, I think this is good for now. 👍